src/storage/src/monitor/monitored_store.rs

```diff
@@ -32,7 +32,7 @@ use crate::{define_state_store_associated_type, StateStore, StateStoreIter};
 /// A state store wrapper for monitoring metrics.
 #[derive(Clone)]
 pub struct MonitoredStateStore<S> {
-    inner: S,
+    pub inner: S,
```

Comment:
We can avoid this `pub`. Refer to Line 292 of this file.

Reply:
> We can avoid this `pub`. Refer to Line 292 of this file.

fixed